rollup.config.js

```diff
@@ -71,7 +69,7 @@ export default {
 
 	server: {
 		input: config.server.input(),
-		output: config.server.output(),
+		output: { ...config.server.output(), sourcemap: !dev },
```

Comment:
I'm realizing now I needed to do the same thing in `client` or else `__sapper__/{dev,build}/client/main.[hash].css` had sourcemaps outnumbering actual CSS by ~6:1. Did you notice this?

Reply:
@babichjacob 
Yeah, I would just have turned CSS source maps off in both client and server output because I've never found the benefits but, I wasn't sure if that would have been too opinionated. 

Turning them off seems to have more upsides like bundle size and a little speed, it may be better to initially have them off in both outputs and direct the user to turn them on if needed in the docs.

What do you think?

Reply:
Moved to [issue 13](https://github.com/babichjacob/sapper-postcss-template/issues/13)